ACCESSIBILITY.md

```diff
@@ -0,0 +1,91 @@
+# Accessibility Policy of The Turing Way Community
+
+Within _The Turing Way_ community, we define accessibility practices as a set of community-wide, behavioral, social, and technical decisions that can be taken to ensure that all are welcome and are able to enter and participate in research, data-driven processes, communities and events despite any barriers for access.
+
+While accessibility is commonly understood with respect to disability, we – meaning _The Turing Way_ community – define accessibility as giving access to everyone, with specific attention given to ensuring proper access for groups of people that encounter different barriers for participation. Barriers may be anything related to, including, and extending beyond the following: 
+- Age
+- Disability
+- Ethnicity and race
+- Family
+- Language (written and/or spoken)
+- Geography
+- Nationality
+- Religion, faith and belief
+- Sex and gender
+- Sexual orientation
+- Socioeconomic status
+- Technical or disciplinary background
+- Mental health diagnoses 
+- Low bandwidth internet access or unstable connectivity (i.e. connection via mobile)
+- Using devices with limited computational resources or capacity
+- Design (ADDING AS PROXY - how do we encompass right-handed bias? Usability of websites AND of physical hardware like computer mice?)
+
+We recognise the complex and interlinking ways in which intersectionality affects identity, and thereby, access or inclusion within any community project like _The Turing Way_. So in this sense, our accessibility practices may overlap with practices that enable inclusion more broadly within any community.
+
+In developing this policy, we aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. This document is a part of a broader collection of documentation across accessibility, listed below:
+- Accessibility policy: Our vision and guiding principles for how we practice accessibility.
```

Comment:
Can we link these? Whichever is currently formed and available.

We can link the rest later.

Reply:
Great call - I'll add those edits in now.